abdm_integrator/hip/tasks.py

```diff
@@ -32,3 +37,12 @@ def process_patient_care_context_link_init_request(self, request_data):
 def process_patient_care_context_link_confirm_request(self, request_data):
     from abdm_integrator.hip.views.care_contexts import GatewayCareContextsLinkConfirmProcessor
     GatewayCareContextsLinkConfirmProcessor(request_data).process_request()
+
+
+@CELERY_PERIODIC_TASK(run_every=crontab(hour='*/2', minute='0'), queue=app_settings.CELERY_QUEUE)
+def process_hip_expired_consents():
+    _process_hip_expired_consents()
```

Comment:
Why use a dummpy method that's not reused and is just one line?

Reply:
Made this way as it was easier to write test cases for `_process_hip_expired_consents` without getting into celery.